src/components/InterfaceEditor.tsx

```diff
@@ -374,9 +421,15 @@ export default ({
         checkInterfaceHasMajorChanges(initialData, parsedInterface)
       ) {
         throw new Error(
-          'Interface cannot have major changes such as updating name, major, type, aggregation, ownership, or editing already existing mappings',
+          'Interface cannot have major changes such as updating name, major, type, aggregation, ownership, or editing sensitive properties of already existing mappings',
         );
       }
+      if (
+        initialData != null &&
+        detectInsensitiveMappingPropertiesChange(initialData, parsedInterface)
```

Comment:
1. I would use a similar naming to `checkInterfaceHasMajorChanges`. We could call this function `checkInterfaceHasMinorChanges`.
2. It would be clearer if you extract the check on the `version_minor` here:
```typescript
if (
  initialData != null &&
  checkInterfaceHasMinorChanges(initialData, parsedInterface) &&
  parsedInterface.minor <= initialData.minor
)
```